dyninstAPI/src/inst-x86.C

```diff
@@ -549,13 +549,17 @@ void registerSpace::initialize()
 int cpuidCall() {
     DWORD result = 0;
 // Note: mov <target> <source>, so backwards from what gnu uses
+#ifdef _WIN64
+	assert(0); // FIXME: Figure out a solution without inline asm.
```

Comment:
I don't think that getting the result of cpuid is going to be feasible without inline assembly, but if there's a Windows API way to do it, go for it. A POSIX method would be even better but I don't expect that to exist.